docs/migration/migrate_8_0.asciidoc

```diff
@@ -338,4 +338,24 @@ The output directory after extracting an archive no longer includes the target p
 *Impact:*
 Configuration management tools and automation will need to be updated to use the new directory.
 
+[float]
+=== `elasticsearch.preserveHost` is no longer valid
+*Details:* The deprecated `elasticsearch.preserveHost` setting in the `kibana.yml` file has been removed.
+
+*Impact:* Configure {kibana-ref}/settings.html#elasticsearch-requestHeadersWhitelist[`elasticsearch.requestHeadersWhitelist`] to whitelist client-side headers.
```

Comment:
question: I know the term is used in the setting name anyway, but we may still want to use another term than whitelist in the text?

Reply:
We are on hold with this effort for the public API. Aren't we? https://github.com/elastic/kibana/issues/71398#issuecomment-658888476
Anyway, it's too late to deprecate them for `v8.0`. We can postpone this work until after 8.0. WDYT?

Reply:
Yea, it's too late. I was just asking if it would make sense to use another term in the `details` section's description. But that's probably not worth it given that the same term is used in the setting's name anyway.